code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -157,7 +157,7 @@ GLOBAL_LIST_INIT(pglass_recipes, list (
 //////////////////////////////
 /obj/item/stack/sheet/plasmaglass
 	name = "plasma glass"
-	desc = "A very strong and very resistant sheet of a plasma-glass alloy."
+	desc = "A very strong and very resistant sheet of a plasma-glass mixture."
```

Comment:
this isn't even a typo, in-fact I'd disagree that this is an improvement at all to this description.

Reply:
_"An alloy is a mixture of chemical elements of which in most cases at least one is a metallic element"_

Neither of the constituents of plasma glass are metals, so it's not an alloy, just a regular mixture.